app/components/createContainer/createcontainer.html

```diff
@@ -98,7 +98,7 @@
           </div>
           <!-- !port-mapping -->
           <!-- access-control -->
-          <div ng-include="'app/components/common/accessControlForm/accessControlForm.html'" ng-if="applicationState.application.authentication"></div>
+          <por-access-control-form form-data="formValues.AccessControlData"></por-access-control-form>
```

Comment:
@WTFKr0 This component takes an extra parameter called `resource-control`. If specified, it will be used to set the default values of the component. If it's undefined, then it will use the default values (see `AccessControlFormData`).

Example:
```
<por-access-control-form form-data="formValues.AccessControlData" resource-control="fromContainer.ResourceControl"></por-access-control-form>
```
You'll need to set the `fromContainer` in the controller (use `ContainerViewModel`).